runtime/src/main/java/io/quarkus/reactivemessaging/http/runtime/WebSocketSink.java

```diff
@@ -54,7 +58,38 @@ class WebSocketSink {
         }
         ssl = WSS.equals(scheme);
 
-        httpClient = vertx.createHttpClient();
+        HttpClientOptions options = new HttpClientOptions();
```

Comment:
Do httpClientOptions and WebClientOptions not share the same interface? Can we reuse this code? It seems pretty similar.

Reply:
in fact they do, very good point, reworked and use utility class to avoid duplication